stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
AFAIU, if we want Stack to compile _explorer_, which is something that we do want - we have a single `stack.yaml` and need to compile all the dependencies in one go.

> It should be handled by a script which builds explorer.

But where would that script be? Seems kind of counter-intuitive that we have a build that builds `cardano-sl` and then we have a script that builds `explorer`. 
We do want to catch any local errors in `explorer` when building, right?